Formula/mage.rb

```diff
@@ -0,0 +1,38 @@
+class Mage < Formula
+  desc "Make/rake-like build tool using Go"
+  homepage "https://magefile.org/"
+  url "https://github.com/magefile/mage.git",
+      :tag => "v1.4.0",
+      :revision => "49bafb86c808d73cabc5353d9ec040745dfab453"
+
+  depends_on "go"
+
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
In theory we could just set `ENV["GOPATH"] = prefix` here and remove the `bin.install buildpath/"bin/mage"` at the end. But not sure if this would be considered good practice style wise?